DrinkingPub/Order.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Vsite.Oom.DrinkingPub
+{
+    internal class Order
+    {
+        public Dictionary<string, int> order;
```

Comment:
Ovaj član treba zaštiti od nekontroliranih promjena izvan klase, a možete ga odmah inicijalizirati tako da ne treba definirati konstruktor:
`private Dictionary<string, int>order = new();`
Za dohvaćanje sadržaja narudžbe, bez mogućnosti promjene izvan klase, može se definirati svojstvo:
`public IEnumerable<KeyValuePair<string, int>> Items =: order.AsEnumerable();`